rclcpp/test/rclcpp/test_node_options.cpp

```diff
@@ -181,6 +181,17 @@ TEST(TestNodeOptions, copy) {
     rclcpp::NodeOptions copied_options = options;
     EXPECT_FALSE(copied_options.use_global_arguments());
     EXPECT_EQ(expected_args, copied_options.arguments());
+
+    EXPECT_EQ(options.enable_rosout(), copied_options.enable_rosout());
+    EXPECT_EQ(options.use_intra_process_comms(), copied_options.use_intra_process_comms());
+    EXPECT_EQ(options.enable_topic_statistics(), copied_options.enable_topic_statistics());
+    EXPECT_EQ(options.start_parameter_services(), copied_options.start_parameter_services());
+    EXPECT_EQ(options.enable_logger_service(), copied_options.enable_logger_service());
+    EXPECT_EQ(options.start_parameter_event_publisher(), copied_options.start_parameter_event_publisher());
+    EXPECT_EQ(options.use_clock_thread(), copied_options.use_clock_thread());
+    EXPECT_EQ(options.allow_undeclared_parameters(), copied_options.allow_undeclared_parameters());
+    EXPECT_EQ(options.automatically_declare_parameters_from_overrides(), copied_options.automatically_declare_parameters_from_overrides());
+
```

Comment:
can you check all attributes here?

Also, this test isn't testing much, because `enable_logger_service_` is `false` by default and the test does not change its value before copy-assigning it. In other words, this test passes even without your fix.

I'd say move this part to the end of this test, create a  `NodeOptions` object, try to set all options to the non-default value, then copy-assign it to a second object, and then assert equality of each attribute.

Reply:
Also, there are some linter failures due to line length, see: https://build.ros2.org/job/Rpr__rclcpp__ubuntu_noble_amd64/351/testReport/junit/rclcpp/uncrustify/test_rclcpp_test_node_options_cpp/

Reply:
You're right, I realized the current test was not tested anything after I submitted the PR.
Thanks for the hint, I will modify the test.